parsec/core/mountpoint/manager.py

```diff
@@ -112,7 +107,10 @@ def get_path_in_mountpoint(
         if timestamp is None:
             workspace = self._get_workspace(workspace_id)
         else:
-            workspace = await self._get_workspace_timestamped(workspace_id, timestamp)
+            try:
+                workspace = self._get_workspace_timestamped(workspace_id, timestamp)
+            except MountpointNotMounted:
+                return await self.remount_workspace_new_timestamp(workspace_id, None, timestamp)
```

Comment:
This is a bit confusing, as it doesn't `remount` but simply `mount` a new workspace, which what this method does.

Reply:
This method can either remount implying it would unmount the workspace at the original timestamp, or only mount a new timestamped workspace at target_timestamp without unmounting anything if the original_timestamp is at None. Better add a comment then, right?